x-pack/test/apm_api_integration/tests/services/throughput.ts

```diff
@@ -200,14 +215,17 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         ).toMatchInline(`"2021-08-03T07:20:10.000Z"`);
       });
 
-      it('has the correct number of buckets', () => {
-        expectSnapshot(throughputResponse.currentPeriod.length).toMatchInline(`91`);
-        expectSnapshot(throughputResponse.previousPeriod.length).toMatchInline(`91`);
+      it('returns same number of buckets for both periods', () => {
+        expect(throughputResponse.currentPeriod.length).to.be(
+          throughputResponse.previousPeriod.length
+        );
       });
 
-      it('has the correct throughput in tpm', () => {
-        expectSnapshot(throughputResponse).toMatch();
-        expectSnapshot(throughputResponse.throughputUnit).toMatchInline(`"second"`);
+      it('calculates the correct throughput avg for both periods', () => {
+        const currentPeriodAvg = mean(throughputResponse.currentPeriod.map((d) => d.y));
+        expectSnapshot(currentPeriodAvg).toMatchInline(`6.46153846153846`);
+        const previousPeriodAvg = mean(throughputResponse.previousPeriod.map((d) => d.y));
+        expectSnapshot(previousPeriodAvg).toMatchInline(`8.50549450549451`);
```

Comment:
As long as we don't have synthetic (rounded) data I don't think this test adds a lot of value. I'm not saying to delete it but we should be aware that it could give a false sense of coverage.